src/components/nonConfiguredComponent/nonConfiguredComponent.css

```diff
@@ -1,4 +1,4 @@
-@import  url('../../../assets/css/_globals.css');
+@import '../../../node_modules/@chartiq/finsemble-ui/react/assets/css/_globals.css';
```

Comment:
Can we do `@import '@chartiq/finsemble-ui/react/assets/css/_globals.css';`  here instead of importing from node_modules?

Reply:
actually wasn't needed at all. looking into this showed me that there were still some global variable scoping issues I had. fixed those.